fabric-api-dev/src/main/java/net/fabricmc/fabric/impl/FabricDevProperties.java

```diff
@@ -0,0 +1,135 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl;
+
+import java.util.function.Supplier;
+
+import com.mojang.brigadier.ParseResults;
+
+import net.minecraft.Bootstrap;
+import net.minecraft.server.command.CommandManager;
+import net.minecraft.server.dedicated.EulaReader;
+import net.minecraft.util.collection.Weight;
+import net.minecraft.world.Heightmap;
+
+import net.fabricmc.loader.api.FabricLoader;
+
+@SuppressWarnings("JavadocReference")
+public class FabricDevProperties {
+	/**
+	 * Logs an error when a weight is set to zero</br>
+	 * Property: <code>fabric.dev.zeroWeightWarning</code></br>
+	 * Default value: true</br>
+	 * {@link Weight#validate(int)}
+	 */
+	public static final boolean ZERO_WEIGHT_WARNING = getProperty("zeroWeightWarning", true);
+
+	/**
+	 * Logs an error when a translation is missing</br>
+	 * Property: <code>fabric.dev.logMissingTranslations</code></br>
+	 * Default value: true</br>
+	 * {@link Bootstrap#logMissing()}
+	 */
+	public static final boolean LOG_MISSING_TRANSLATIONS = getProperty("logMissingTranslations", true);
+
+	/**
+	 * Logs an error if Block classes don't end with Block and if Item classes don't end with Item</br>
+	 * Property: <code>fabric.dev.logConventionIssues</code></br>
+	 * Default value: true</br>
+	 * {@link net.minecraft.block.Block#Block} and {@link net.minecraft.item.Item#Item}
+	 */
+	public static final boolean LOG_CONVENTION_ISSUES = getProperty("logConventionIssues", true);
+
+	/**
+	 * Skips creating the eula.txt file and always agrees to the EULA</br>
+	 * <strong>Note: By enabling this, you declare that you have agreed to the EULA.</strong></br>
+	 * Property: <code>fabric.dev.alwaysAgreeToEula</code></br>
+	 * Default value: false</br>
+	 * {@link net.minecraft.server.dedicated.EulaReader#EulaReader} and {@link EulaReader#createEulaFile()}
+	 */
+	public static final boolean ALWAYS_AGREE_TO_EULA = getProperty("alwaysAgreeToEula", false);
+
+	/**
+	 * Registers Minecraft's debug commands
+	 * (TestCommand, RaidCommand, DebugPathCommand, DebugMobSpawningCommand,
+	 * WardenSpawnTrackerCommand, SpawnArmorTrimsCommand, ServerPackCommand),
+	 * and if on the server DebugConfigCommand</br>
+	 * Property: <code>fabric.dev.registerDebugCommands</code></br>
+	 * Default value: true</br>
+	 * {@link CommandManager#CommandManager}
+	 */
+	public static final boolean REGISTER_DEBUG_COMMANDS = getProperty("registerDebugCommands", true);
+
+	/**
+	 * Logs an error if a command threw an exception</br>
+	 * Property: <code>fabric.dev.enableCommandExceptionLogging</code></br>
+	 * Default value: true</br>
+	 * {@link CommandManager#execute(ParseResults, String)}
+	 */
+	public static final boolean ENABLE_COMMAND_EXCEPTION_LOGGING = getProperty("enableCommandExceptionLogging", true);
```

Comment:
I wonder if these should be named such as `fabric.dev.command.logging.exception` (with all of the others following the same pattern?